tests/integrations/coprocessor/test_select.rs

```diff
@@ -2018,15 +2019,19 @@ fn test_rc_read() {
     assert_eq!(row_count, expected_data.len());
 }
 
-#[test]
+#[cfg(feature = "failpoints")]
```

Comment:
Could you move it to tests/failpoints folder? Otherwise, it may interfere with other concurrent running tests.